### DISCUSSION TITLE
Inaccurate Korean Translation in the GitHub Mobile App

### DISCUSSION THREAD

**Discussion by wdkdot** [community/community#128516] [Mobile] [answered]
Hello,

I would like to report an incorrect translation in the GitHub mobile app. When viewing a user's profile or repository, there is a button in the top right corner labeled "Report" which allows users to report abuse or spam. However, in the Korean language setting, this button is labeled as "보고서" which means "report" as in a document or report.

The correct translation for the "Report" button in this context should be "신고" which means to report an issue or abuse. 

Thank you :)

![image](https://github.com/community/community/assets/76516426/a6e37fae-80af-415c-879d-5343d26881a6)

add)
![339660875-93d85832-a8f6-49b1-9d9b-f084a161a0cd](https://github.com/community/community/assets/76516426/a1eff9b4-a186-4e5a-bdc2-f5e0f549cba1)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 버전 1.165.0(133064262)
Device: iPhone 15 Pro Max (iOS 18.0)
</details>

**Comment by taki-on** [comment, marked as answer]
@greenbean1210 oh my god thank you for reporting the issue. We will fix it as soon as possible 🙇 I believe 신고 should be the correct translation as you suggested!
